src/dbcp/data_mart/counties.py

```diff
@@ -181,7 +181,7 @@ def _get_existing_plant_attributes(engine: sa.engine.Engine) -> pd.DataFrame:
     #         plant_id_eia,
     #         generator_id,
     #         count(fuel_type_code_pudl) as fuel_type_count
-    #     FROM "data_warehouse"."mcoe"
+    #     FROM "data_warehouse"."pudl_generators"
```

Comment:
The table formerly known as `mcoe` was renamed in PUDL because it didn't fully compute MCOE and is really a table of generators and all it's attributes. I renamed the data warehouse table to be more consistent with what is actually in the table.